connector/src/main/scala/org/apache/spark/sql/cassandra/CassandraSourceRelation.scala

```diff
@@ -558,7 +557,7 @@ object CassandraSourceRelation extends Logging {
 
   val SearchPredicateOptimizationRatioParam = ConfigParameter[Double] (
     name = "spark.sql.dse.search.autoRatio",
-    section = DseReferenceSection,
+    section = ReferenceSection,
```

Comment:
So, just to make sure I have this straight... we're keeping the "dse" bits in the name even though we're going out of our way to move the docs for these props out of sections that are clearly tied to DSE.  Is this... just a backwards-compatibility issue?

Reply:
I think we should change all of the names on these parameters, I was going to do a big rename ticket later.